<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>871</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>ajouter une reclamation</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>911</width>
        <height>481</height>
       </rect>
      </property>
      <property name="title">
       <string>Reclamation</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Type de Reclamation :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="cinr">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="type_de_reclamation">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>881</width>
         <height>331</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tabWidgetPage1">
        <attribute name="title">
         <string>Client</string>
        </attribute>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>50</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Cin de client</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="cin_de_client">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>50</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="sat_de_client">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>100</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="feed_back">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>150</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>45</x>
           <y>100</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Satisfaction</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>155</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>feed back</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_ajouter_avie">
         <property name="geometry">
          <rect>
           <x>640</x>
           <y>250</y>
           <width>151</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter l'avie d'un client </string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>salle</string>
        </attribute>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>40</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>ID SALLE</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="id_salle">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>40</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>90</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>inconvinent</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="inconv">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>90</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_ajouter_salle">
         <property name="geometry">
          <rect>
           <x>492</x>
           <y>220</y>
           <width>151</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>ajouter l'inconvinent</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonplay">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>210</y>
           <width>91</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>PLAY VID</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonstop">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>210</y>
           <width>91</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>STOP VID</string>
         </property>
        </widget>
        <widget class="QVideoWidget" name="widgetvd" native="true">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>20</y>
           <width>331</width>
           <height>151</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tabWidgetPage2">
        <attribute name="title">
         <string>Employee</string>
        </attribute>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>40</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>CIN d employee abcent</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="cin_emp_ab">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>40</y>
           <width>191</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>90</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>justification</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="justi">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>90</y>
           <width>191</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_ajouter_employee">
         <property name="geometry">
          <rect>
           <x>452</x>
           <y>210</y>
           <width>161</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>ajouter l'employee</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>sponsor</string>
        </attribute>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>40</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>90</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>raison </string>
         </property>
        </widget>
        <widget class="QLineEdit" name="nom_sp">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>40</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="rais_sp">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>80</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_ajouter_spons">
         <property name="geometry">
          <rect>
           <x>442</x>
           <y>180</y>
           <width>131</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>ajouter le sponsor</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>40</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>CODE de reclamation</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="code_r">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>affichage</string>
     </attribute>
     <widget class="QTableView" name="tablerec">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>21</y>
        <width>821</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="solved">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>420</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TRI SOLVED</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="type">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>460</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TRI TYPE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PDF">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>440</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stat_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>480</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>stat</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>supprimer</string>
     </attribute>
     <widget class="QLineEdit" name="rec_supp">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_delete_rec">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>180</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>176</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CODE DE REC</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>modify</string>
     </attribute>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CODR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>200</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CAS</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mody_code">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="mody_cas">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>190</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modify">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>280</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>modify</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>chercher</string>
     </attribute>
     <widget class="QTableView" name="affchercher">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>210</y>
        <width>711</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>chercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="chercher_r">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_chercher">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>80</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>chercher</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
